src/back/bootstrap/index.js

```diff
@@ -9,10 +9,18 @@ const
 
 module.exports = (app) => {
     db(formatMongooseDbUriFromEnv(env));
+    extendRequest();
     preMiddleware(app);
     return app.use(controllers);
 };
 
+function extendRequest() {
+    const authService = require('../controllers/auth/auth.service'),
+        http = require('http');
+    http.IncomingMessage.prototype.getCurrentUser = authService.getCurrentUser;
```

Comment:
Extending third-party prototype might be a quick win, but in general doesn't look like a good idea. Have you thought about a middleware which extends request instances rather that extending prototype?